### ISSUE TITLE
[1.12.2] Kitchen Counter change colour

### ISSUE THREAD

**Issue by ProNinja21** [MrCrayfish/MrCrayfishFurnitureMod#561]
I downloaded the mod into RLcrafts modpack and when i place a counter of any colour it changes it to white. Is that because of something in the modpack or is it a bug wuth my mod download?

Please complete the following information:
- Minecraft Version:1.12.2
- Mod Version:6..30

**Comment by AryaRaiin** [comment]
Same here!

Not consistent with which blocks change color. Haven't had issues with kitchen counter colors. Placing curtains and modern chairs shows the same issue for me. Change in color isn't consistent.

The curtains I placed were grey. Both couches should be yellow. Hwyla sees the curtains as magenta, and both couches as being yellow.

In this picture, I broke the curtain next to the couches and the pink one block updated, causing it turn yellow.

Let me know if you want some more help/info. This is in one of my modpacks and also happening on other clients.

![oddcolored](https://user-images.githubusercontent.com/24833872/79676822-c5b4d580-819e-11ea-8b61-9fd0b6a5a17b.png)

Minecraft 1.12.2
Version : 6.3.1
Forge 14.23.5.2854
